### DISCUSSION TITLE
Who will be behind Ibax?  ///  İbax ın arkasında kimler var?

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#270] [Ideas]
Herkes İbax ın ne kadar iyi proje olduğundan bahsediyor açıkcası Buna ne kadar çok inanmak istesemde gözüme çarpan bazı durumlar var bunları Söyle sıralamak gerekirse.
1-)  İbax ın arkasında kim var Destekcileri kimlerdir bunları göremiyorum 
2-)  İbax telegram veya Discord Adminleri Yetrince aktif değil  Bir soru sorulduğunda Cevap bile gelmiyor ve daha projenin başlarındayız

Bu ikisi en temel sorularım. Ben ibax ın  neden bu kadar sevildiğini güvenildiğini gerçekden anlamamış durumdayım bir bilginiz varsa benimle paylaşırsanız minnettar kalırım...
Herkese iyi günler--

**Comment by byblckd** [comment]
Olay tam anlamıyla sosyal medya gücü başka açıklması yok.Proje daha çok taze ama bilirsiniz ki bu coğrafya da yaşayan herkes abartmayı çok sever. İbax da bunun  eseri

**Reply by ghost** [reply]
Sana katılıyorum kanka

**Comment by ghost** [comment]
Şaun Yutupçular felaket punp lamış durumda projeyi bakalım öylemi öyleyse biz burdayız 👍
